index.html

```diff
@@ -2053,8 +2059,8 @@ <h4 id="signed-raw">Signed Raw Transaction</h4>
           if (cachedUTXOs.length) { 
             // If the collateral is missing (spent, or switched wallet) then remove the current MN
             if (localStorage.getItem("masternode")) {
-              const strColla = JSON.parse(localStorage.getItem("masternode")).collateralTxId;
-              if (!cachedUTXOs.find(cUTXO => cUTXO.id === strColla)) {
+              const { collateralTxId, outidx } = JSON.parse(localStorage.getItem("masternode"));
+              if (!cachedUTXOs.find(cUTXO => cUTXO.id === collateralTxId && cUTXO.vout === outidx)) {
```

Comment:
Could you replace this with our `isMasternodeUTXO` function?
e.g; I believe this should work: `.find(isMasternodeUTXO)`